### PR TITLE
Update LightburnSoftware.Lightburn 1.2.00

### PR DESCRIPTION
**Pull request by dwittenberg**
[Update LightburnSoftware.Lightburn 1.2.00](https://github.com/microsoft/winget-pkgs/commit/f20f50aed149ff6a3356ed122e448586ac56645b)

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65135)

### CODE REVIEW

File: manifests/l/LightburnSoftware/Lightburn/1.2.00/LightburnSoftware.Lightburn.locale.en-US.yaml
Comment (ItzLevvie) [inline manifests/l/LightburnSoftware/Lightburn/1.2.00/LightburnSoftware.Lightburn.locale.en-US.yaml:12]:
```suggestion
PackageName: LightBurn
```
This must match Control Panel for consistency.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> When writhing here in CamelCase, i must rename all Files. Shall I?

You just need to add all suggestions as batches and then commit them.

Since there's already a `LightburnSoftware` folder changing the capitalization will cause two folders with the same name.

If you want to do this then you will need to delete all versions, then wait until it's merged, and then re-add all of the versions with the correct capitalization i.e. `LightBurnSoftware`

File: manifests/l/LightburnSoftware/Lightburn/1.2.00/LightburnSoftware.Lightburn.locale.en-US.yaml
Comment (dwittenberg) [reply manifests/l/LightburnSoftware/Lightburn/1.2.00/LightburnSoftware.Lightburn.locale.en-US.yaml:23]: Ok, thanks.

**Pull request closed without merging**
